src/current/v24.1/molt-fetch.md

```diff
@@ -324,13 +504,13 @@ You can use the `cdc_cursor` value with an external change data capture (CDC) to
 
 ## Examples
 
-The following examples demonstrate how to issue `molt fetch` commands to load data onto CockroachDB. 
+The following examples demonstrate how to issue `molt fetch` commands to load data into CockroachDB. 
```

Comment:
```suggestion
The following examples demonstrate how to issue `molt fetch` commands to import data into CockroachDB. 
```
I think previously we called it importing. I wonder if loading is somehow different. I notice that the subsections below also use "Load".

Reply:
"Load" is generally used in migration scenarios. My concern about "import" is that it could be conflated with the `IMPORT INTO` command, which is only one of the methods we can apply here.